docs/source/en/model_doc/xlm-prophetnet.mdx

```diff
@@ -49,20 +49,105 @@ The Authors' code can be found [here](https://github.com/microsoft/ProphetNet).
 
 ## XLMProphetNetModel
```

Comment:
The examples that were moved in the doc file were for the tokenizers, where the example are not standardized to work well with the Copied from statements. It should not be necessary for the model which contain standardized examples (and you have properly replaced the checkpoints in all of them!)

Reply:
That makes sense. So, do you recommend not having any examples in the doc file?

Reply:
Not for models no.

Reply:
Thanks for the clarification. Done.